<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraView</class>
 <widget class="QWidget" name="CameraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_launch_camera">
        <property name="text">
         <string>Launch camera</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_take_picture">
        <property name="text">
         <string>Take a Picture</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_save_image">
        <property name="text">
         <string>Save Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabContainer">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_camera">
      <attribute name="title">
       <string>Camera</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_screen">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
